documentation/trouble_session_logging.markdown

```diff
@@ -0,0 +1,78 @@
+---
+title: "PuppetDB 3.2 » Troubleshooting » Session Logging"
+layout: default
+canonical: "/puppetdb/latest/trouble_session_logging"
```

Comment:
This needs an `.html` extensions, like so `/puppetdb/latest/trouble_session_logging.html`.